packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -15,10 +15,15 @@ const modules = require('../../config/modules');
 const pkgJson = require(paths.appPackageJson);
 const bpkReactScriptsConfig = pkgJson['backpack-react-scripts'] || {};
 
-const includePrefixes = bpkReactScriptsConfig.babelIncludePrefixes || [];
-includePrefixes.unshift('bpk-');
-includePrefixes.unshift('@skyscanner/bpk-');
-includePrefixes.unshift('@skyscanner/backpack-web/bpk-mixins');
+const includePrefixes = bpkReactScriptsConfig.jestIncludePrefixes || [];
```

Comment:
We need a separate list for Jest as its module resolution mechanism is more strict compared to Webpack, but I still want to pass dependencies by their names instead of using bulk prefixes